config-files/jhtdb-config.json

```diff
@@ -55,7 +55,12 @@
       "name": "Position",
       "description": null,
       "cardinality": 3
-    }
+    },
+    {
+      "code": "n",
+      "name": "eddyviscosity",
+      "description": null,
+      "cardinality": 1  
```

Comment:
There's a `}` missing before the `],` to close the new variable addition

Reply:
I'll fix it over the weekend.

Reply:
fixed in github